.github/workflows/build-electron-production.yml

```diff
@@ -26,12 +25,12 @@ jobs:
         env:
           KEYS_DECRYPT_PASSPHRASE: ${{ secrets.KEYS_DECRYPT_PASSPHRASE }}
       - name: Build/release Electron app
-        uses: samuelmeuli/action-electron-builder@v1
+        uses: coparse-inc/action-electron-builder@v1.0.0
```

Comment:
for me: do you know why the old electron builder stopped working?

Reply:
See [original comment](https://github.com/neopostmodern/wohnungsbot/pull/157#issue-1833070346) above – it didn't technically stop working.